src/mixed-line-bar-chart/utils.ts

```diff
@@ -95,9 +109,13 @@ export function calculateOffsetMaps(
       if (y < 0) {
         const lastValue = lastMap?.negativeOffsets.get(key) || 0;
         map.negativeOffsets.set(key, lastValue + y);
+        setMinValue(key, lastValue + y);
+        setMaxValue(key, lastValue + y);
       } else {
         const lastValue = lastMap?.positiveOffsets.get(key) || 0;
         map.positiveOffsets.set(key, lastValue + y);
+        setMinValue(key, lastValue + y);
+        setMaxValue(key, lastValue + y);
       }
```

Comment:
Don't you see any issues when repeating `lastValue + y` 6 times here?

Reply:
I will refactor this utility, resolving the thread.